src/styles/dark-theme.css

```
@@ -18,6 +18,6 @@
   --color-css: hsl(209 70% 24%);
   --color-js: hsl(49 74% 23%);
   --color-tools: hsl(122 56% 23%);
-  --color-recipes: hsla(276 39% 32%);
-  --color-svg: hsl(347 23% 31%);
+  --color-recipes: hsla(276, 39%, 32%);
```

```suggestion
  --color-recipes: hsl(276, 39%, 32%);
```

Тут вроде закралась ошибка из прошлого... Я делал 🐥